internal/db/migrations/2024-06-27.0-ingest_payments_2.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+
+ALTER TABLE ingest_payments
+  ADD COLUMN src_asset_type text NOT NULL,
+  ADD COLUMN dest_asset_type text NOT NULL,
```

Comment:
We are safe, but if we have entries in this table this migration will fail.

Reply:
Indeed! I also tried explaining that in the **Known limitations** section.